h5p_proplist.go

```diff
@@ -9,16 +9,20 @@ package hdf5
 // #include <string.h>
 // inline static
 // hid_t _go_hdf5_H5P_DEFAULT() { return H5P_DEFAULT; }
+// hid_t _go_hdf5_H5P_DATASET_CREATE() { return H5P_DATASET_CREATE; }
 import "C"
 
+import "unsafe"
+
 type PropType C.hid_t
 
 type PropList struct {
 	Identifier
 }
 
 var (
-	P_DEFAULT *PropList = newPropList(C._go_hdf5_H5P_DEFAULT())
+	P_DEFAULT          *PropList = newPropList(C._go_hdf5_H5P_DEFAULT())
+	H5P_DATASET_CREATE PropType  = PropType(C._go_hdf5_H5P_DATASET_CREATE()) // Properties for dataset creation
```

Comment:
to be consistent, I believe this should be named `P_DATASET_CREATE`.

Reply:
Fixed